auth/src/App.js

```diff
@@ -4,17 +4,24 @@ const bodyParser = require("body-parser");
 const morgan = require("morgan");
 const { sendResponse } = require("./utils/sendResponse");
 const router = require("./api/index");
+const RateLimit = require("express-rate-limit");
 
 const app = express();
 require("dotenv").config();
 
+const limiter = new RateLimit({
+  windowMs: 1 * 60 * 1000,
+  max: 5,
+});
+
```

Comment:
What's the reasoning behind 5 requests per minute? I'm just trying to think if there's a case we'd go over that. Like login, verify, get saved resources? I'm not sure how often we call verify

Reply:
Just the default that the documentation provided. I think it's pretty suitable, not sure how much we need to play around with it unless you think we should increase it